### DISCUSSION TITLE
How to make FanControl start with system?

### DISCUSSION THREAD

**Discussion by StilettoStyle** [Rem0o/FanControl.Releases#932] [Q&A]
Hello, im having problem with program not starting when i start windows, i have option ticked in FanControl but its not working. 
Am i missing something?
thanks in advance

Edit:
I managed to "fix" it by addint shortcut to startup folder, but im not sure thats how its ment to be?

**Comment by EdwardKiefer** [comment]
There should be an entry in Task scheduler "FanControl"
